### ISSUE TITLE
Malware infection attempt through https://www.laravelgis.com/

### ISSUE THREAD

**Issue by ArjunKishore** [opengis/laravel-gis#1]
Looks like your site has malware injection, please look into it

**Comment by opengis** [comment]
Hello,

Thanks for pointing this out, it should be temporary fixed now (I haven't found the root cause for this yet but I deactivated registration until I have time to investigate this). It looks like a privilege escalation vulnerability though. If you have any more details, please let me know.

Thanks again!

**Comment by opengis** [comment]
I finally found the vulnerability: a Livewire function to delete a post was accessible publicly. Fortunately, I use soft delete and I have good backups. It should now be fixed and I haven't found any other vulnerability that can affect users (like XSS). Again, thank you for warning me. I will close the issue but if you spot anything else, do not hesitate to open a new one.

**Comment by ArjunKishore** [comment]
Thanks for actioning this quickly. Please check the link below or search the site in virustota.com l to get more information about malware. Looks the vulnerability still exist

Beside that, thanks for your article and package, its super useful,
https://www.virustotal.com/gui/url/d5999c065c6d6456c041976734cfc324732b8ba2c6aae93046a5bb9498b4fe04

**Comment by opengis** [comment]
It's like ESET doesn't refresh on Virustotal, I launch a scan with 91 security providers and barely get 20 queries in the reverse proxy server logs :(
I also manually analysed the source of every page, logged in and not logged in, I ran Quttera, Sucuri and Sitelock scans and didn't find any more vulnerability or malware. I will keep monitoring this more closely in the next days.

Will also try to find time to add new content shortly.
